lib/PRC/Controller/User.pm

```diff
@@ -226,6 +229,9 @@ sub settings :Path('/settings') :Args(0) {
       my $selected_langs = $languages_form->values->{lang_select};
       $user->update_langs($selected_langs);
       $c->stash->{alert_success} = 'Your preferred languages are updated.';
```

Comment:
Can we convert these to session variables? Now that we are adding a reload, stash will be cleared by the time user lands on the page & the message will be missing. Session variable will be there for a little longer. Something like `$c->session->{alert_success}` in `sub skip_confirm`.